public/app/percona/shared/services/advisors/Advisors.utils.ts

```diff
@@ -4,14 +4,19 @@ export const groupAdvisorsIntoCategories = (advisors: Advisor[]): CategorizedAdv
   const result: CategorizedAdvisor = {};
 
   advisors.forEach((advisor) => {
-    const { category, summary } = advisor;
+    const { category, summary, checks } = advisor;
+
+    const modifiedChecks = checks.map((check) => ({
+      ...check,
+      disabled: check.disabled ? true : false,
```

Comment:
Why is there a need for this?

Reply:
Because backend is not sending disabled field which filter needs to be working properly.

Reply:
Oh, right, then you can use `!!check.disabled` to get a boolean value of an truthy/falsy one.